taxi/models.py

```diff
@@ -30,7 +30,12 @@ def get_absolute_url(self):
 
 class Car(models.Model):
     model = models.CharField(max_length=255)
-    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="cars",
+        null=False
+    )
```

Comment:
The `manufacturer` field in the `Car` model is correctly defined as a `ForeignKey` to the `Manufacturer` model. Ensure that the `Manufacturer` model exists and is correctly defined, as it is referenced here.